src/grpc.ts

```diff
@@ -141,7 +151,27 @@ export default class GrpcClient {
 
         return { request, limit }
     }
+    
+    public buildSearchStreamRequest(
+        customer_id: string,
+        query: string,
+    ): BuildSearchStreamRequestResponse {
+        const request = new SearchGoogleAdsStreamRequest()
+        request.setCustomerId(customer_id)
+        request.setQuery(query)
+
+        const has_limit = query.toLowerCase().includes(' limit ')
+        let limit = 0 // The default limit is 0, which means no limit.
+        if (has_limit) {
+            limit = +query
+                .toLowerCase()
+                .split(' limit ')[1]
+                .trim()
+        }
 
+        return { request, limit }
```

Comment:
The caller of this function doesn't seem to use the `limit` part of this return?

Reply:
Removed from the return